types/react-json-pretty/index.d.ts

```diff
@@ -11,10 +12,15 @@ export as namespace JSONPretty;
 export = JSONPretty;
 
 declare const JSONPretty: JSONPretty;
-type JSONPretty = ComponentClass<JSONPretty.JSONPrettyProps>;
+
+type JSONPretty = ComponentClass < JSONPretty.JSONPrettyProps >;
 
 declare namespace JSONPretty {
-    interface JSONPrettyProps extends HTMLProps<JSONPretty> {
+    interface JSONPrettyProps extends HTMLProps < JSONPretty > {
         json: {} | string;
+        themeClassName?: string;
+        replacer
```

Comment:
The indentation here makes this harder to read. `replacer?(key: string, value: any): string;` would be preferred.